addons/binding/pom.xml

```diff
@@ -30,6 +30,8 @@
     <module>org.openhab.binding.exec</module>
     <module>org.openhab.binding.feed</module>
     <module>org.openhab.binding.feed.test</module>
+    <module>org.openhab.binding.feed.transform</module>
```

Comment:
Should it maybe live add `org.openhab.transform.feed` && `org.openhab.transform.feed.test`